homeassistant/components/solarlog/quality_scale.yaml

```diff
@@ -0,0 +1,69 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
```

Comment:
We should remove this test code
https://github.com/home-assistant/core/blob/25aea140be4a159d7006a4e703d23c102f50d043/tests/components/solarlog/test_config_flow.py#L47-L51